pkg/csr/csr.go

```diff
@@ -193,9 +193,8 @@ func (a *Approver) validateNodeName(nodeName string) (bool, error) {
 	// check if the node name matches the lookup of any of the instance addresses
 	hasEntry, err := matchesDNS(nodeName, windowsInstances)
 	if err != nil {
-		return false, fmt.Errorf("unable to map node name to the addresses of Windows instances: %w", err)
-	}
-	if hasEntry {
+		a.log.Info("error occurred with reverse DNS lookup, falling back to hostname validation", "error", err)
```

Comment:
```suggestion
		a.log.Info("reverse DNS lookup not available, falling back to hostname validation", "error", err)
```

I would spin it toward a info-like msg